src/main/java/teammates/ui/template/FeedbackSubmissionEditQuestion.java

```diff
@@ -27,6 +29,7 @@ public FeedbackSubmissionEditQuestion(FeedbackQuestionAttributes questionAttribu
         questionText = questionAttributes.getQuestionDetails().questionText;
         visibilityMessages = questionAttributes.getVisibilityMessage();
         questionType = questionAttributes.questionType;
+        giverType = questionAttributes.giverType;
```

Comment:
I don't think this <code>FeedbackSubmissionEditQuestion</code> needs to know the question's giver type. It only needs to know if the message needs to be displayed, so just passing that in is sufficient.